client/signup/steps/design-picker/style.scss

```diff
@@ -21,6 +22,14 @@
 			padding-left: 96px;
 			padding-right: 96px;
 		}
+
+		&.design-picker-step__is-large-thumbnails {
+			.design-picker__grid {
+				grid-template-columns: repeat( auto-fill, minmax( 450px, auto ) );
```

Comment:
I think we need to consider the size on the mobile

Reply:
🤔 when I tested it does work on mobile, the thumbnail shrinks down so that it fits.

But I'm not entirely happy with when it switches between 1 and 2 columns though. I think I'll change my approach so that it explicitly has 2 columns on desktop and 1 column on phones, rather than trying to find a magic number where the column breaking works the way I want it to.

Reply:
But when I test on chrome, the width is too wide and overflows the screen 🤔

Reply:
Ah great! I wasn't testing in Chrome, thanks for pointing that out!